### ISSUE TITLE
Training ST-GCN Model For Untrimmed, Multi-Class Videos In Custom Data Set :

### ISSUE THREAD

**Issue by lakshaymehra** [open-mmlab/mmskeleton#357]
Hi,

Thank you for this wonderful repository. 

I wanted to ask you about generating an annotation file similar to https://github.com/open-mmlab/mmskeleton/blob/master/resource/category_annotation_example.json for my own data set. 

I have a lot of long, untrimmed videos in the data set with each video containing multiple actions belonging to different classes.

1. Is multi-class Skeleton Based Action Recognition possible using MMSkeleton? Or should the videos only contain 1 action each which would imply that there is only 1 class(or category) per video?
2. If multi-class Skeleton Based Action Recognition is possible, Can you share an example of the format or how the category_annotation_example.json file should look like for multiple actions in one video belonging to different classes(or categories)?
3. What are the other changes needed to the code base for training ST-GCN model on such a data set?

Looking forward to your response,

Thank You!

**Comment by yjxiong** [comment]
Currently the classifiers in this repo are all implemented as 1 vs. N  SoftMax. To enable multi-label recognition, you can change the loss function to the corresponding type. Sorry we don't have example for this use case.
